### PR TITLE
MR-2844 Rates Across Subs should use radio buttons

### PR DESCRIPTION
**Pull request by haworku**
## Summary
Use radio buttons instead of checkbox on rates across subs. Also add design tweaks for summary page work.

#### Related issues
https://qmacbis.atlassian.net/browse/MR-2844
See design tweak comments on https://qmacbis.atlassian.net/browse/MR-2686

#### Screenshots
Screen Shot 2022-12-05 at 5 15 13 PM
Screen Shot 2022-12-05 at 5 12 25 PM

#### Test cases covered
- No new tests, had to fix tests to work for the radio buttons
- Our tests are really long in RateDetails, we need to simplify. Felt like this was out scope for this ticket but this is also the test we have periodic timeouts in unit tests because we are doing a lot. 

## QA guidance
- There should be no change in the data stored or behaviors on edit. The radio button is UI only and not stored as a field in db, we just store the programs list in the associated dropdown that appears when you click "Yes".
- Validations state work checking, for yes, no, yes with the dropdown selecton, yes with dropdown.

### CODE REVIEW

**Review by JasonLin0991 — CHANGES_REQUESTED** [review]
1️⃣ Bug. 1️⃣ Styling. Other wise looks good.

**Review by JasonLin0991 — CHANGES_REQUESTED** [review]
See above. I'm pretty sure I selected `Requested changes` and not `Comments`.

File: services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx [hunk 1]
```diff
@@ -737,39 +729,79 @@ export const RateDetails = ({
...
+                                                                            {getIn(
+                                                                                errors,
+                                                                                `rateInfos.${index}.hasSharedRateCert`
+                                                                            )}
+                                                                        </PoliteErrorMessage>
+                                                                    )}
+                                                                    <FieldRadio
+                                                                        id={`hasSharedRateCertYes-${index}`}
+                                                                        name={`rateInfos.${index}.hasSharedRateCert`}
+                                                                        label="Yes"
```
[the remark this answers is not in the document]
Comment (haworku) [reply services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx:789]: Thanks, addressed in [de08cfc](https://github.com/CMSgov/managed-care-review/pull/1364/commits/de08cfc993000b2ceffe4f918b788d28a9699b51)

**Review by JasonLin0991 — APPROVED** [review]
Thanks for the bug fix!

**Pull request merged**